tests/fixtures/anim/matrices/matrix_aln_lengths_1.tab

```diff
@@ -0,0 +1,4 @@
+	MGV-GENOME-0264574:1	MGV-GENOME-0266457:2	OP073605:3
+MGV-GENOME-0264574:1	39253	39169	39147
+MGV-GENOME-0266457:2	39176	39594	39572
+OP073605:3	39150	39568	57793
```

Comment:
I think you need to use the more common ``.tsv`` extension rather than ``.tab`` for GitHub to automatically render these as tables.